pixi/src/pixi.hbs

```diff
@@ -1,5 +1,71 @@
 {{#htmlWebpackPlugin.files.js }}
-<amp-script layout="container" src="{{ . }}" sandbox="allow-forms">
-  <button class="ap-a-btn">Hello amp-script!</button>
-</amp-script>
+  <amp-script class="ap-t-pixi-checks" src="{{ . }}" layout="container" sandbox="allow-forms">
+
+    <div class="ap-m-input-bar">
+      <input id="input-field" class="ap-m-input-bar-field" type="url" name="url" placeholder="Your URL">
+      <button id="input-submit" class="ap-m-input-bar-submit" name="button">
+        Analyze
+      </button>
+    </div>
+
+    <div class="ap-t-pixi-checks-primary">
+      <div id="LARGEST_CONTENTFUL_PAINT_MS" class="ap-t-pixi-checks-primary-metric">
+        <h3 class="ap-t-pixi-checks-primary-metric-title">Loading speed | LCP
+          <br>
+          <label>Analyzing..</label>
```

Comment:
nit: This shouldn't say `Analyzing...` before the user has pressed the button. Does this section need to be shown at all before the user presses the button? Or can we use form submission event to display this section?

Reply:
Absolutely! Ferdi already worked out a solution for that using skeleton placeholders (should be on zeplin). I will implement this in a separate pr.